src/instana/util/traceutils.py

```diff
@@ -24,8 +24,11 @@ def get_active_tracer() -> Optional[InstanaTracer]:
     try:
         # ToDo: Might have to add additional stuff when testing with async and tornado tracer
         current_span = get_current_span()
-        if current_span and current_span.is_recording():
-            return tracer
+        if current_span:
```

Comment:
This valuates to false because the span isn't activated in the current context. Reference: [link](https://opentelemetry.io/docs/languages/python/cookbook/#manually-setting-span-context).

<img width="1073" alt="Screenshot 2024-09-09 at 2 47 03 PM" src="https://github.com/user-attachments/assets/d3da16bd-a9d3-48ac-a79f-2d416e0374b1">